README.md

```diff
@@ -147,6 +148,7 @@ Name | Description
 Name | Type | Description
 --- | --- | ---
 [redhatci.ocp.junit2dict]() | Filter | Transforms a JUnit into a dictionary
+[redhatci.ocp.junit2obj]() | Filter | Transforms a JUnit into a corresponding JSON text (will replace `redhatci.ocp.junit2dict`)
```

Comment:
Could you please remove this entry with references with `redhatci.ocp.junit2dict` in https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/510#issuecomment-2553454615 was mentioned that the expectation is to remove the plugin rather than replace it by junit2obj.